### ISSUE TITLE
type="password"

### ISSUE THREAD

**Issue by lrkwz** [grnadav/databind#9]
What about binding passwords?

```
Uncaught NoModificationAllowedError: Failed to set the 'innerText' property on 'HTMLElement': The 'input' element does not support text insertion.
```

**Comment by grnadav** [comment]
thats a bug. input of type password should be modified using the 'value' attribute, not the innerText one.
(_weird inconsistency in the spec :-/ _ edit: nope, just me)

**Comment by grnadav** [comment]
Fixed in 0.3.2. Closing
